draft-ietf-scitt-architecture.md

```diff
@@ -267,15 +263,10 @@ The Statement is considered opaque to Transparency Service, and MAY be encrypted
 
 Subject:
 
-: This term has the same definition as in {{RFC8392}}, which relies on the definition in RFC7519.
-The `sub` (subject) claim identifies the principal that is the subject of the CWT.
-The claims in a CWT are normally statements about the subject.
-In SCITT, `sub` identifies the entity about which statements, and receipts are made.
-The subject value MUST either be scoped to be locally unique in the context of the Issuer or be globally unique.
-The processing of this claim is generally application specific.
-The `sub` value is a case-sensitive string containing a StringOrURI value.
-Issuers use `sub` to identify the entity about which they are making Signed Statements.
-Transparency Services use `sub` to identify the entity about which they are issuing a Receipt.
+: an identifier that represents the organization, device, user, entity or Artifact about which Statements (and Receipts) are made and by which a logical collection of Statements can be grouped.
```

Comment:
```suggestion
: an identifier that represents the organization, device, user, entity or Artifact about which Statements (and Receipts) are made and by which a logical collection of Statements, Receipts and addition relevant media can be grouped.
```

Reply:
Attempting to communicate that a subject can resolve to a collection of media, including but not limited to statements and receipts.

Reply:
Can you elaborate a bit. The subject is on the header of a statement. A statement payload can be anything, including a receipt that is re-registered. But, it's still a statement.

Reply:
its less to do with "subject" which is a text string... and more to do with "logical collection of..." which is limited to specific media types or is not... current text suggests its limited, I disagree with that.

Reply:
I withdraw the section.